src/utils.tsx

```diff
@@ -85,3 +85,24 @@ export const extractNumber = (text: string): number | null => {
   }
   return null;
 };
+
+type MakReviewersURLParams = {
+  apiHost?: string | null;
+  reviewersHost?: string | null;
+  url: string;
+  useInsecureProxy?: boolean;
+};
+
+export const makeReviewersURL = ({
```

Comment:
That function is confusing. I guess it is because the API returns a download URL with the API host instead of the reviewers host?

I would suggest to not do that because we should not involve the `apiHost` in this function I believe. I think we should simply make the download URL relative when `useInsecureProxy` is set to `true` and prepend this relative URL with the `reviewersHost` when `useInsecureProxy` is `false`.

That seems simpler and more future-proof (if, for instance, the API starts to return download URLs with the reviewers host, would this function continue to do its job right?).

WDYT?

Reply:
Yes, that sounds like a good idea.